CHANGELOG.md

```diff
@@ -10,6 +10,18 @@ The classical shadows documentation has been improved (including a tutorial!) by
 
 In terms of additions, a new type of benchmark quantum circuits, "rotated" randomized benchmarking (RB) quantum circuits have been added by @Misty-W, for more general benchmarks.
 
+```py
+import numpy as np
+from mitiq.benchmarks import generate_rotated_rb_circuits
+
+circuits = generate_rotated_rb_circuits(
+    n_qubits=2,
+    num_cliffords=10,
+    theta=2 * np.pi * np.random.Generator.random(),
+    trials=100,
```

Comment:
In the simplest form, we only need `n_qubits` and `num_cliffords`, and I like the idea of advertising the easiest way to use something. On the flip, I totally understand wanting to advertise the full capability of something newly released. So fine by me either way.
```suggestion
```

Reply:
Thanks @natestemen and sorry for the delay in responding! I'm going to keep this as-is, since the "rotated" part is what we want to highlight. Always good to think about the simplest way to represent something though.